cmd/juju/controller/listmodels_test.go

```diff
@@ -265,6 +265,23 @@ func (s *BaseModelsSuite) TestModelsOwner(c *gc.C) {
 	s.checkAPICalls(c, "BestAPIVersion", "ListModels", "ModelInfo", "Close")
 }
 
+// TestModelsForAdmin tests that a model admin user will get model credential.
+// Credential will only appear in non-tabular format - either yaml or json.
+func (s *BaseModelsSuite) TestModelsWithCredentials(c *gc.C) {
+	for i, infoResult := range s.api.infos {
+		// let's say only some models will have credentials returned from api...
```

Comment:
do we also want to have this? I'm fine with a test that ensure if we *do* chose to omit the tag, things don't break.

Reply:
Since there is a potential not to have this information, I'd rather leave this test as is.